WordPress/UITests/Utils/XCTest+Extensions.swift

```diff
@@ -16,6 +16,8 @@ extension XCTestCase {
         continueAfterFailure = false
 
         app.launchArguments = ["-wpcom-api-base-url", WireMock.URL().absoluteString, "-no-animations", "-ui-testing", "-logout-at-launch"]
+        // Reset scenarios for Wiremock's stateful behavior
```

Comment:
👋 @mokagio, would appreciate it if you could help review, especially for changes on this file. The changes work, but I feel the code could be better/cleaner, just not sure how exactly.

Reply:
I'd also appreciate @mokagio's thoughts on this part but my first thought is that the functions could live in a WireMock helper file, maybe? 

Another thing is that if `fetchScenarios` and `resetScenarios` were called something like `fetchWireMockScenarios` and `resetWireMockScenarios` we probably wouldn't need the commented line?

Reply:
Hey folks! Sorry for the long reply time... Just wanted to let you know that I'll be looking at this shortly.

Reply:
@jostnes @tiagomar 

> Another thing is that if `fetchScenarios` and `resetScenarios` were called something like `fetchWireMockScenarios` and `resetWireMockScenarios` we probably wouldn't need the commented line?

Building on top of that, it should be possible to move these utils in UITestsFoundation [`WireMock`](https://github.com/wordpress-mobile/WordPress-iOS/blob/0ac167921891bd904d358e7472be65fbe757c308/WordPress/UITestsFoundation/WireMock.swift). 

With that, we might do something like `WireMock.resetScenarios()` or `WireMock.reset(<# scenario name #>)` if we go down https://github.com/wordpress-mobile/WordPress-iOS/pull/21288/#pullrequestreview-1572823086.